src/pages/home/ReportScreen.js

```diff
@@ -273,6 +270,8 @@ class ReportScreen extends React.Component {
 
         const policy = this.props.policies[`${ONYXKEYS.COLLECTION.POLICY}${this.props.report.policyID}`];
 
+        const isTopMostReportId = Navigation.getTopmostReportId() === getReportID(this.props.route);
```

Comment:
Coming from https://github.com/Expensify/App/issues/24555

Relying on a side-effectful procedure (even when the side effect is returning the current state of something) is not allowed in React's `render` and can lead to various problems, one of which is what we observed in https://github.com/Expensify/App/issues/24555.